src/tools/schema2kotlin.ts

```diff
@@ -101,11 +101,13 @@ class KotlinGenerator implements ClassGenerator {
   fields: string[] = [];
   fieldVals: string[] = [];
   setFields: string[] = [];
-  fieldSets: string[] = [];
   encode: string[] = [];
   decode: string[] = [];
   fieldsReset: string[] = [];
   getUnsetFields: string[] = [];
+  fieldsForCopyDecl: string[] = [];
```

Comment:
Either in this PR or a near future one: [Destructuring declarations](https://kotlinlang.org/docs/reference/multi-declarations.html) (`component1`, `component2`, etc.) should also be implemented. At least, we should do this if we want to be more in like with [Kotlin data class](https://kotlinlang.org/docs/reference/data-classes.html#data-classes-and-destructuring-declarations) expectations.